CHANGELOG.md

```diff
@@ -92,6 +92,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Samuel Giddins](https://github.com/segiddins)
   [#8451](https://github.com/CocoaPods/CocoaPods/issues/8451)
 
+* Fixed test host delegate methods to not warn about unused arguments.
```

Comment:
`.  ` with two trailing spaces

Reply:
Fixed! Thank you @segiddins. Could you please merge https://github.com/CocoaPods/cocoapods-integration-specs/pull/214 first so I can also update the submodule ref? Or is there a different process for this?

Reply:
@jmkk done. please update it so this gets green